src/parse/pegutil.js

```diff
@@ -3,8 +3,20 @@
 
 import * as types from '../common/types'
 
+export function buildLogicalExpression (head, tail) {

```

Comment:
I'm guessing this was needed because of the extra whitespace? I think if you include those in the token (see other comment) you can use `buildBinaryExpression` instead.

Reply:
See above comment resolving this